config/schema.yaml

```diff
@@ -694,6 +655,66 @@ schema:
         type: string
         doc: A string value composed only of letters, numbers, and underscores.
 
+  dataUnitRegions:
+    -
```

Comment:
Is there a reason why this is a list rather than associative array like `dataUnits` and `dataUnitJoins`?

Reply:
I felt like the conceptual key for one of these items is the "relates" entry, which is itself a list and hence can't be used as a key directly.  So a list seemed like the next-best thing.